csm/core/blogic/const.py

```diff
@@ -931,11 +931,16 @@
 CREATE_USER_OPERATION = 'CREATE_USER'
 GET_USER_OPERATION = 'GET_USER'
 DELETE_USER_OPERATION = 'DELETE_USER'
-RGW_CLIENT_ERROR_MSG = 'Can not process the request'
+MODIFY_USER_OPERATION = 'MODIFY_USER'
+RGW_CLIENT_ERROR_MSG = 'Unable to process the request'
 CREATE_KEY_OPERATION = 'CREATE_KEY'
 REMOVE_KEY_OPERATION = 'REMOVE_KEY'
+STD_ERROR_CODES = {
+    408:{"Code":"RequestTimeout"},
+    503:{"Code":"S3ServiceUnavailable"}
+    }
```

Comment:
Minor: please, fix the format. Add spaces, move curly brackets, etc.

btw, the idea of handling error codes this way is very good.